src/data/equiv/basic.lean

```diff
@@ -975,3 +975,40 @@ protected def congr_right {r r' : setoid α}
   (eq : ∀a₁ a₂, @setoid.r α r a₁ a₂ ↔ @setoid.r α r' a₁ a₂) : quotient r ≃ quotient r' :=
 quot.congr_right eq
 end quotient
+
+/-- If a function is a bijection between `univ` and a set `s` in the target type, it induces an
+equivalence between the original type and the type `↑s`. -/
+noncomputable def set.bij_on.equiv {α : Type*} {β : Type*} {s : set β} {f : α → β}
+  (h : set.bij_on f set.univ s) : α ≃ s :=
```

Comment:
Pretty printer will print it as `set.bij_on.equiv _`. I'd turn `f` into an explicit argument.

Reply:
Done.